composer.json

```diff
@@ -18,13 +18,19 @@
         "issues": "https://github.com/stwe/DatatablesBundle/issues"
     },
     "require": {
-        "php": ">=7.1",
-        "symfony/framework-bundle": "^3.0|^4.0",
+        "php": ">=7.2",
```

Comment:
Why not keeping PHP 7.1 as a minimum requirement?
I think we should follow the minimum version required by Symfony.

Reply:
I'm sorry, requiring 7.2 is a bit strict. I will change this!